modules/tink_cc/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/tink-crypto/tink-cc",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
+            "name": "No Maintainer Specified"
```

Comment:
@morambro Do you mind listing yourself as a maintainer for this BCR module?

Reply:
Done